openedx/features/pakx/lms/overrides/tasks.py

```diff
@@ -125,9 +127,12 @@ def check_and_send_email_to_course_learners():
         data = {'course_name': course_overview.display_name,
                 'username': item.user.username, 'email': item.user.email,
                 'language': get_user_preference(item.user, LANGUAGE_KEY),
-                'completed': grades.percent >= 1.0,
-                'status_message': "Completed" if grades.percent >= 1.0 else "Pending"}
-        if grades.percent >= 1.0:
+                'completed': grades.passed and float(get_course_progress_percentage(create_dummy_request(Site.objects.get_current(), item.user), text_type(item.course_id))) >= 100,
```

Comment:
store completion status in a variable and use it at both places
```
course_progress = float(get_course_progress_percentage(create_dummy_request(Site.objects.get_current(), item.user), text_type(item.course_id)))

completed = grades.passed and course_progress >= 100
data['course_progress'] = course_progress
```
and log course_progress in the email sending function.

Reply:
Done!